ramls/mod-users-bl.raml

```diff
@@ -134,10 +185,15 @@ resourceTypes:
     /{id}:
       type: { compositeUserResource: { "typeName" : "id" } }
       is: [includeable]
+      delete:
```

Comment:
What responses are expected from a delete operation (there don't appear to be any documented here)?

Reply:
I remodelled the response to be more explicit now.

Before it was modelled as an optional HTTP method of a compositeUserResource, which was rather implicit.

Reply:
Thanks